documentation/getting-started-cli/source/debugging-with-gdb-lldb.rst

```diff
@@ -149,31 +149,49 @@ Any LLDB arguments can be specified, as normal.
 The support library provides some extra commands and specialized
 summarizers for commonly-encountered Dylan objects.
 
+Note that if you're using the Boehm GC, which is the default on Unix systems,
+it is necessary to tell ``lldb`` not to stop on the signals used by the GC::
+
+  (lldb) process handle -p yes -s no -n no SIGPWR
```

Comment:
The actual signals used varies from platform to platform; I'll look up what it is on non-Linux platforms.

Reply:
FreeBSD uses SIGUSR1 and SIGUSR2. I don't think macOS (Darwin) uses any signals.

Reply:
Done, thanks.